Marlin/src/sd/cardreader.cpp

```diff
@@ -389,7 +389,7 @@ void CardReader::mount() {
 #endif
 
 void CardReader::manage_media() {
-  static uint8_t prev_stat = TERN(INIT_SDCARD_ON_BOOT, 2, 0);
+  static uint8_t prev_stat = 2;       // First call, no prior state
```

Comment:
This call might make it possible to receive a recovery prompt when first mounting a card, IF if it possible to configure the board to not mount on boot.

On my SKR 1.3 it seems to mount on boot automatically, and I didn't see an obvious way to prevent that. This was true even though `INIT_SDCARD_ON_BOOT` evaluates to false. I think that constant has just become inconsistent and obsolete in past refactoring.

Reply:
I think you are correct. The original purpose was to deal with the code of `manage_media` being called from `MarlinUI:update`, where now it is better handled by a call from `idle` which shouldn't get called until `setup` is completed.

Reply:
On second examination, there is some minor purpose for the value 2 versus the value 0. Basically, if the initial value is 0 there will be a message on the screen that the media was inserted on first boot. That is somewhat the expected behavior for an SD card inserted into an external LCD, and with "`INIT_SDCARD_ON_BOOT`" the idea was to suppress the message under some conditions, such as when the SD card is always inserted.

Maybe this behavior is obsolete. There was also some behavior around the LCD being detected, which was retained because it seemed related to potentially an extra init of the SD card. But all of this may be obsolete with the simplified SD / media management as part of the `idle` loop.